integration-test/helpers/mocks.webkit.js

```diff
@@ -288,30 +383,31 @@ async function withMockedWebkit (page, mocks) {
                 postMessage: async (data) => {
                     /** @type {MockCall} */
                     const call = [msgName, data, response]
+                    let thisResponse = response
                     window.__playwright.mocks.calls.push(JSON.parse(JSON.stringify(call)))
 
-                    // If `data.messageHandling.methodName` exists, this means we're trying to use encryption
-                    // therefor we mimic what happens on the native side by calling the relevant window method
-                    // with the encrypted data
-                    const fn = window[data.messageHandling.methodName]
-                    if (typeof fn === 'function') {
-                        // @ts-ignore
-                        fn(encryptResponse(data, response))
-                        return
-                    }
-
```

Comment:
I've moved this after the check for `isMulti` so we can use multiple responses on Catalina as well.